requirements.txt

```diff
@@ -1,2 +1,3 @@
 tornado >= 4.0.1
-motor >= 0.4.1
\ No newline at end of file
+motor >= 0.4.1
+lxml >= 3.4.4
```

Comment:
Adding lxml adds system requirements (e.g. libxml2, libxslt). Please use built-in `xml.etree` (https://docs.python.org/3/library/xml.etree.elementtree.html). lxml have better performance but we don't need it in tests.